packages/mui-joy/src/ToggleButtonGroup/ToggleButtonGroupContext.tsx

```diff
@@ -0,0 +1,20 @@
+'use client';
+import * as React from 'react';
+
+interface IToggleButtonGroupContext {
+  onClick: (
+    event: React.MouseEvent<HTMLButtonElement>,
+    childValue: readonly string[] | string | number | undefined,
+  ) => void;
+  value?: string | (string | number)[] | null;
```

Comment:
We can reuse ToggleButtonGroup value:
```suggestion
  value?: ToggleButtonGroupProps['value'];
```